NetKet/Machine/pylayer.hpp

```diff
@@ -65,6 +65,22 @@ void AddLayerModule(py::module &m) {
                                                                    "SumOutput")
         .def(py::init<int>(), py::arg("input_size")) ADDLAYERMETHODS(LayerType);
   }
+  {
```

Comment:
I believe you also would need to change the python unit tests, especially `Test/Machine/test_machine.py`, since in there I was using the old activation!

Reply:
No wait, how is it possible that the tests are passing?!